recipes/app1.rb

```diff
@@ -26,13 +26,13 @@
 
 sudo 'openid-staging' do
   user 'openid-staging'
-  commands sudo_commands('openid-staging-unicorn', 'openid-staging-delayed-job')
+  commands osl_sudo_commands('openid-staging-unicorn', 'openid-staging-delayed-job')
   nopasswd true
 end
 
 sudo 'openid-production' do
   user 'openid-production'
-  commands sudo_commands('openid-production-unicorn',
+  commands osl_sudo_commands('openid-production-unicorn',
                          'openid-production-delayed-job')
```

Comment:
Let's just do it this way as it's cleaner.
```suggestion
  commands osl_sudo_commands('openid-production-unicorn', 'openid-production-delayed-job')
```